tavern/_core/dict_util.py

```
@@ -484,42 +484,74 @@ def _format_err(which):
-                                e_val, current_response_val, keys + [i], strict
-                            )
-                        except exceptions.KeyMismatchError:
-                            # Doesn't match what we're looking for
-                            logger.debug(
-                                "%s did not match next response value %s",
-                                e_val,
-                                current_response_val,
-                            )
+                            try:
+                                check_keys_match_recursive(
+                                    e_val, current_response_val, keys + [i], strict
```

Ruff (0.16.2)

[warning] Consider `[*keys, i]` instead of concatenation

Replace with `[*keys, i]`

(RUF005)